src/amchart/CommonFunnel.js

```diff
@@ -1,7 +1,7 @@
 "use strict";
 (function(root, factory) {
     if (typeof define === "function" && define.amd) {
-        define(["d3", "../common/HTMLWidget", "amcharts.funnel", "require"], factory);
+        define(["d3", "../common/HTMLWidget", "amcharts.funnel", "require", "amcharts.plugins.export"], factory);
```

Comment:
@GordonSmith i dont follow what your saying there? amcharts.plugins.export exports on the global amcharts variable